BSPConvertLib/Source/EntityConverter.cs

```diff
@@ -566,17 +572,6 @@ private void GiveWeaponOnStartTouch(Entity trigger, Entity weaponEnt)
 			trigger.connections.Add(connection);
 
 			GiveWeaponAmmoOnStartTouch(trigger, weaponEnt);
-
-			var targets = GetTargetEntities(weaponEnt); //TODO: more robust solution for entities targeting other entities inside a trigger_multiple
```

Comment:
Shouldn't you remove the `GetTargetEntities` code in `ConvertInitTrigger` as well?

Reply:
forgot that was there, yep.